### PR TITLE
Migrate LevelGroup LevelSource data.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: bin/oneoff/data_fix/level_group_data [hunk 1]
```diff
@@ -134,7 +173,8 @@ def main
     user_level, level_source_data = get_user_level_and_level_source_data(
       user_level_id
     )
-    next unless user_level && level_source_data
```
Comment (Hamms) [inline bin/oneoff/data_fix/level_group_data:137]: why was this line removed?

Comment (ashercodeorg) [reply bin/oneoff/data_fix/level_group_data:137]: As `get_user_level_and_level_source_data` was changed to throw an exception (in some prior version, it did not).
